packages/react-vapor/src/components/form/Form.tsx

```diff
@@ -0,0 +1,32 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+type FormMods = 'mod-header-padding';
+
+export interface IFormProps {
+    title?: string;
+    className?: string | string[];
+    children?: React.ReactNode;

```

Comment:
If you define the Form as a React.FunctionComponent, you won't have to specify the children prop type.